zerver/lib/test_fixtures.py

```diff
@@ -143,7 +190,14 @@ def is_template_database_current(
         settings_hash_status = all([check_setting_hash(setting_name, status_dir)
                                     for setting_name in check_settings])
         hash_status = files_hash_status and settings_hash_status
+        migration_op = what_to_do_with_migrations(migration_status, settings=settings)
+
+        if not hash_status or migration_op == 'scrap':
```

Comment:
Let's make `if not hash_status` a separate if statement.

Reply:
ok